.github/workflows/static_analysis.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         shell: bash
         # yamllint disable rule:line-length
         run: |
-          Rscript -e "install.packages(c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl', 'roxygen2', 'testthat'), repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())"
+          Rscript ./.ci/install-r-deps.R --build --roxygen
```

Comment:
Notice... `{testthat}` and all its dependencies were being installed unnecessarily here! So this change should also make the static analysis job a little faster 🎉